# Message published by the action server
float64 x  # x-coordinate of the robot
float64 y  # y-coordinate of the robot

float64 vel_x  # x-velocity of the robot
float64 vel_z  # z-velocity of the robot


